### ISSUE TITLE
Add setTypeface method (or mPaintText accessor) to TitlePageIndicator

### ISSUE THREAD

**Issue by ai212983** [JakeWharton/ViewPagerIndicator#39]
May I suggest adding setTypeface method to TitlePageIndicator class? I'm using something like this right now:

``````
 ```
    public void setTypeface(Typeface typeface) {
        mPaintText.setTypeface(typeface);
    }
 ```
``````

Having mPaintText accessor will do the trick as well.

**Comment by JakeWharton** [comment]
Done in `dev` branch.

**Comment by JakeWharton** [comment]
Released with 2.2.1.
